src/app.js

```diff
@@ -1 +1,37 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+setInterval(() => {
+  const time = new Date();
+  let hours = time.getHours();
+  let minutes = time.getMinutes();
+  let seconds = time.getSeconds();
+
+  if (hours < 10) {
+    hours = `0${hours}`;
+  }
+
+  if (minutes < 10) {
+    minutes = `0${minutes}`;
+  }
+
+  if (seconds < 10) {
+    seconds = `0${seconds}`;
+  }
```

Comment:
padStart would do the same job for you:
`let seconds = time.getSeconds().toString().padStart(2, '0');`

Reply:
👍